wandb/sdk/interface/interface.py

```diff
@@ -766,10 +766,17 @@ def deliver_run_start(self, run_pb: pb.RunRecord) -> MailboxHandle:
         run_start.run.CopyFrom(run_pb)
         return self._deliver_run_start(run_start)
 
-    def publish_launch_wandb_config_parameters(
-        self, include_paths: List[List[str]], exclude_paths: List[List[str]]
+    def deliver_job_input(
```

Comment:
why do you want it to be deliver and not just publish? do want to get some response. 
Basically, we have 2 modes: 
- "publish_" which is one directional communication (only send)
- "deliver_" which is bidirectional communication (send and wait on response)

Trying to understand why you want it to be deliver type

Reply:
Oh I just misunderstood the distinction. I will change it back to publish